e2e/tests/contracts.rs

```diff
@@ -1959,17 +1948,33 @@ async fn setup_node_with_high_price() -> Result<Vec<Wallet>> {
         fee_params: fee_parameters,
         ..Default::default()
     });
-    let node_config = Some(NodeConfig {
+
+    let mut node_config = NodeConfig {
         starting_gas_price: 1100,
         ..NodeConfig::default()
-    });
+    };
+
+    if historical_execution {
+        // The temp dir will be deleted at the end of this scope
```

Comment:
perhaps you want to return the `temp_dir` as part of this function then? i'm curious why the tests are passing if the directory is dropped :?

Reply:
https://github.com/FuelLabs/fuels-rs/pull/1628/commits/15809ae8fb0954af18c2170eb9ed4bbc38d89739